### PR TITLE
Update CSV metadata

### PR DESCRIPTION
**Pull request by andreasgerstmayr**
* add metadata.annotations.description 1-liner description
* list operator features in the spec.description field
* add link to spec.description where to find more information about tempo
* add pre-requisites for using the operator in the description field (object storage)
* "TempoStack is the Schema for the tempostacks API." -> "TempoStack is the spec for Tempo deployments."
* update "provided by": "Grafana Tempo SIG Operator"
* change support field from "Tempo Operator SIG" to "Grafana Tempo SIG Operator"
* update keywords

Resolves: #444

### CODE REVIEW

File: bundle/community/manifests/tempo-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -33,18 +33,20 @@ metadata:
...
     containerImage: ghcr.io/grafana/tempo-operator/tempo-operator
-    createdAt: "2023-06-06T14:36:33Z"
+    createdAt: "2023-06-19T15:32:10Z"
+    description: Create and manage deployments of Tempo, a high-scale distributed
+      tracing backend.
     operators.operatorframework.io/builder: operator-sdk-v1.27.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/grafana/tempo-operator
-    support: Tempo Operator SIG
+    support: Grafana Tempo SIG Operator
```
Comment (pavolloffay) [inline bundle/community/manifests/tempo-operator.clusterserviceversion.yaml:42]: Grafana Tempo Operator SIG

File: bundle/community/manifests/tempo-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -501,10 +503,22 @@ spec:
...
   description: |
-    Tempo is an open source, easy-to-use, and high-scale distributed tracing backend, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki.
-    It can ingest common open source tracing protocols including Jaeger, Zipkin, and OpenTelemetry.
+    Tempo is an open source, easy-to-use, and high-scale distributed tracing backend.
+    It can ingest common open source tracing protocols including Jaeger, Zipkin, and OpenTelemetry and requires only object storage to operate.
+    Please refer to the [Tempo documentation](https://grafana.com/docs/tempo/latest/) for more information about Tempo.
 
     The Community Tempo Operator manages Tempo deployments in Microservices mode.
+
+    ### Operator features
```
Comment (pavolloffay) [inline bundle/community/manifests/tempo-operator.clusterserviceversion.yaml:512]: We should add the authn/authz to the list as well

File: bundle/community/manifests/tempo-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (andreasgerstmayr) [reply bundle/community/manifests/tempo-operator.clusterserviceversion.yaml]: :+1: done

File: bundle/community/manifests/tempo-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (andreasgerstmayr) [reply bundle/community/manifests/tempo-operator.clusterserviceversion.yaml:512]: I've added:
**Authentication and Authorization**: Access to Tempo can be secured with RBAC

and I also copied the feature list to the readme.

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**